packages/platforms/electron-player/src/renderer/src/classes/ElectronMachine.ts

```diff
@@ -0,0 +1,112 @@
+import { Machine, DeviceInfo } from '@castmill/device';
+
+// get the version of the electron app from
+
+export class ElectronMachine implements Machine {
+  /**
+   * Since a browser does not support a MAC address or another
+   * persistent GUID, we will use a random UUID.
+   *
+   */
+  async getMachineGUID(): Promise<string> {
+    //TODO use a real hardware identifier
```

Comment:
I have some code for this in our old electron player: https://github.com/OptimalBits/castmill-electron/blob/master/src/services/device.ts

Reply:
Implemented this as in the old player. the guid is now the sha1 hash of the first mac address